Assets/Code/src/Runtime/Match/Systems/TestTriggerSystem.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Unity.Entities;
+using Unity.Physics;
+using Unity.Physics.Systems;
+using Unity.Jobs;
+using Unity.Transforms;
+
```

Comment:
Remove double space. Add namepace. At the very minimum, put it into the HouraiTeahouse.FantasyCrescendo namespace. Please move all other types not in this namespace to this one.